lang/system/pl.xml

```diff
@@ -326,26 +573,250 @@ System Vita3K jest gotowy!</completed_setup>
+Jest też głównie polecenana dla graficznym interfejsie użytkownika.</firmware_font_package_description>
+			<theme_background>Motyw i tło</theme_background>
+			<current_theme_content_id>Identyfikator obecnego motywu:</current_theme_content_id>
+			<reset_default_theme>Resetuj domyślny motyw</reset_default_theme>
+			<using_theme_background>Obecnie używane tło motywu</using_theme_background>
+			<clean_user_backgrounds>Wyczyść tła użytkownika</clean_user_backgrounds>
+			<current_start_background>Obecne tło startowe:</current_start_background>
+			<reset_start_background>Restuj tło startowe</reset_start_background>
+			<background_alpha>Tło Alpha</background_alpha>
+			<select_background_alpha>Definiuje preferowaną przeźroczystość tła.
+Wartość minimalna jest nieprzeźroczysta, a wartość minimalna jest przeźroczysta.</select_background_alpha>
+			<delay_background>Opóźnienie dla zmiany teł</delay_background>
+			<select_delay_background>Definiuje opóźnienie (w sekundach) przed zmianą teł.</select_delay_background>
+			<delay_start>Opoźnienie dla ekranu startowego</delay_start>
+			<select_delay_start>Definiuje opóźnienie (w sekundach) przed powrotem do ekranu startowego.</select_delay_start>
+		</gui>
 		<network name="Sieć">
+			<psn_signed_in>Zalogowany do PSN</psn_signed_in>
+			<psn_signed_in_description>Jeśli zaznaczone, gry będą zakładać że użytkownik jest połączony z PlayStation Network (ale w trybie offline).</psn_signed_in_description>
+			<enable_http>Włącz HTTP</enable_http>
+			<enable_http_description>Zaznacz, aby umożliwić grom korzystanie z protokołu HTTP w internecie.</enable_http_description>
+			<timeout_attempts>Próby przekroczenia czasu połączenia HTTP</timeout_attempts>
+			<timeout_attempts_description>Ilość prób połączenia do wykonania, gdy serwer nie odpowiada.
+Może być użyteczne, jeśli posiadasz NIESTABILNE lub BARDZO WOLNE połączenie internetowe.</timeout_attempts_description>
+			<timeout_sleep>Uśpienie przekroczenia czasu połączenia HTTP</timeout_sleep>
+			<timeout_sleep_description>Nakłada uśpienie między próbami połączenia z serwerem, gdy nie odpowiada.
+Może być użyteczne, jeśli posiadasz NIESTABILNE lub BARDZO WOLNE połączenie internetowe.</timeout_sleep_description>
+			<read_end_attempts>Próby zakończenia odczytu HTTP</read_end_attempts>
+			<read_end_attempts_description>Ilość prób do wykonania, gdy nie ma więcej danych do odczytu,
+niższa wartość może poprawić wydajność, ale może sprawić, że gry będą niestabilne, jeśli masz naprawdę słabe połączenie z internetem.</read_end_attempts_description>
+			<read_end_sleep>Uśpienie przy próbach zakończenia odczytu HTTP</read_end_sleep>
+			<read_end_sleep_description>Nakłada uśpienie, gdy nie ma więcej danych do odczytu,
+niższa wartość może poprawić wydajność, ale może sprawić, że gry będą niestabilne, jeśli masz naprawdę słabe połączenie z internetem.</read_end_sleep_description>
 		</network>
-		<close>Zamykać</close>
+		<debug name="Debugowanie">
```

Comment:
also here

Reply:
@Croden1999 This one doesn't have a name so I removed this one before, rebasing 13 commits did a thing or two. Will fix, but menubar thing should stay.